connectors-element-template-links.txt

```diff
@@ -50,6 +51,7 @@ https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/azu
 https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/blueprism
 https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/easy-post
 https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/email
+https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/embeddings-vector-db
```

Comment:
I don't think this connector is available in 8.7, is it? I only see it in the docs for 8.8. Can you confirm? @ztefanie

Reply:
Yes, you are right, it is only available in 8.8

Reply:
Fixed it on connectors side with this PR: https://github.com/camunda/connectors/pull/4790

Reply:
Just to clarify the way forward as we'll have the same issue with the Agentic AI connectors: are we linking to 8.8 now and revert back to `docs/` without a version number after 8.8 was released?

Reply:
@maff feel free to add you thoughts to this ticket: https://github.com/camunda/connectors/issues/4791